### ISSUE TITLE
[🇻🇳] Vietnamese language support

### ISSUE THREAD

**Issue by lehuukhoa** [spacehuhn/esp8266_deauther#809]
I've created vietnamese language support for the web interface.
I think I did not understand how to update the program with the new language.
I added the string to webfile.h . Don't work.

**Comment by tobozo** [comment]
please see how other translations were integrated and submit a PR so we can merge this as a new language

**Comment by bukkit188** [comment]
Sorry but the language code for Vietnamese is Vi or Vie, not VN, according to Wikipedia.
Hope it help!

**Comment by tobozo** [comment]
@bukkit188 you're right, instead of using *Country codes* as language indicators, the issue titles should be using *Locales*.

Unfortunately there aren't any unicode Emojis for languages/locales, so we're using the Emoji Country codes to *flag* the issues.

Of course the translation file will be named `vi.lang` after the Vietnamese iso639-1 language code, just [like the other files](https://github.com/spacehuhn/esp8266_deauther/tree/master/web_interface/lang).
